packages/datagrid/src/components/DefaultHeaderRenderer/MenuDropDownQualityBar.component.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Overlay, Popover, Button } from 'react-bootstrap';
+import { Action } from '@talend/react-components';
+import './MenuDropDownQualityBar.scss';
+
+
+export default function MenuDropDownQualityBar(props) {
+	const [isOpen, setIsOpen] = React.useState(false);
+	const buttonRef = React.useRef();
+	return (<>
+		<Action
+			buttonRef={buttonRef}
+			className={'button-menu-dropdown'}
```

Comment:
I guess we want to use css module for this class instead of having a global in the scss file